_pages/community.html

```diff
@@ -1,187 +1,285 @@
+      </div>
+
+      <div class="link-item" id="crystalshards.org">
+        <a class="link-item__main" href="https://crystalshards.org">
+          {% include icons/manage-search.svg %}
+          crystalshards.org
+        </a>
+        <p>Lists shard repositories on GitHub with dependencies.</p>
+      </div>
+
+      <div class="link-item" id="awesome">
+        <a class="link-item__main" href="https://github.com/veelenga/awesome-crystal">
+          {% include icons/manage-search.svg %}
+          awesome-crystal
+        </a>
+        <p>A curated Awesome List of Crystal shards, tools and tutorials.</p>
+      </div>
+    </div>
+
+
+    <div class="link-items">
+      <h3>Tooling</h3>
+
+      <div class="link-item">
+        <a class="link-item__main" href="https://github.com/veelenga/awesome-crystal#editor-plugins">
+          {% include icons/extensions.svg %}
+          Editor Plugins
+        </a>
+        <p>Compilation of plugins for source editors and IDEs.</p>
+      </div>
+
+      <div class="link-item">
+        <a class="link-item__main" href="https://crystal-ameba.github.io/">
+          {% include icons/ameba.svg %}
+          Ameba
+        </a>
+        <p>Code style linter for Crystal</p>
+      </div>
+    </div>
+  </div>
+
+</section>
+
+<h2>Crystal Sphere</h2>
+
+<section>
+
+  <div class="topics">
+    <div class="link-items">
+      <h3>Podcasts & Videos</h3>
+
+      <div class="link-item">
+        <a class="link-item__main" href="https://podcast.chicagocrystal.org/">
+          {% include icons/podcasts.svg %}
+          Chicago Crystal Podcast
+        </a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://www.youtube.com/channel/UCI1RvHPG6S9mw4eRoJfH2kA">
+          {% include icons/subscriptions.svg %}
+          Chicago Crystal Videos</a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://www.youtube.com/playlist?list=PLe1-A91ZPTpAIO9YRZbAlPT3l1S5pMPrX">
+          {% include icons/video-library.svg %}
+          Crystal 1.0 Conference Playlist
+        </a>
+      </div>
+    </div>
+    <div class="link-items">
+      <h3>Blogs</h3>
+
+      <div class="link-item">
+        <a class="link-item__main" href="https://dev.to/t/crystal">
+          {% include icons/dev.to.svg %}
+          Crystal on dev.to</a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://medium.com/tag/crystal-lang">
+          {% include icons/medium.svg %}
+          Crystal on Medium</a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://lbarasti.com/tags/crystal/">
+          {% include icons/newsmode.svg %}
+          Lorenzo Barasti</a>
+      </div>
+      <div class="link-item">
+        <a class="link-item__main" href="https://manas.tech/blog/">
+          {% include icons/manas.svg %}
+          Manas.Tech Blog</a>
+      </div>
```

Comment:
This is not Crystal specific

Reply:
Not necessarily, yes. Still a blog we can recommend, thogh 🤷 I don't mind removing it here, though.

Reply:
https://manas.tech/blog/tagged/crystal/ maybe?

Reply:
and it might be stupid, but we need to link the `/blog` page too!